providers/catalog.go

```diff
@@ -1215,4 +1216,29 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	// Zuora Configuration
+	Zuora: {
+		AuthType: Oauth2,
+		BaseURL:  "https://{{.subdomain}}.zuora.com",
+		OauthOpts: OauthOpts{
+			GrantType:                 ClientCredentials,
+			AuthURL:                   "https://rest.test.zuora.com/oauth/auth_mock",
+			TokenURL:                  "https://rest.test.zuora.com/oauth/token",
```

Comment:
I think that these need the substitution as well

Reply:
updated the URL's